eng/scripts/CodeCheck.ps1

```diff
@@ -130,6 +130,11 @@ try {
         & $dotnet run -p "$repoRoot/eng/tools/BaselineGenerator/"
     }
 
+    Write-Host "Re-generating blazor files"
```

Comment:
@SteveSandersonMS we're looking at a solution where the the blazor files are checked in to the source and we add a check, the same way we verify ref assemblies are up to date, to ensure these files are up to date. It is slightly ugly to check in the files, but it does make reasoning about these files a bit easier.

What do you think?